ui.frontend/src/view/FormFieldBase.js

```diff
@@ -194,11 +194,21 @@ class FormFieldBase extends FormField {
     } 
 
     #syncAriaDescribedBy() {
+        let ariaDescribedby = '';
         let widgetElement = typeof this.getWidget === 'function' ? this.getWidget() : null;
         let widgetElements = typeof this.getWidgets === 'function' ? this.getWidgets() : null;
         widgetElement = widgetElements || widgetElement;
         if (widgetElement) {
-            widgetElement.setAttribute('aria-describedby', `${this.getId()}__errormessage ${this.getId()}__shortdescription ${this.getId()}__longdescription`);
+           if (this.getDescription()) {
+            ariaDescribedby += `${this.getId()}__longdescription`
+          }
+           if (this.getTooltipDiv()) {
+            ariaDescribedby += ` ${this.getId()}__shortdescription`;
+          }
+           if (this.getErrorDiv() && this.getErrorDiv().innerHTML) {
```

Comment:
Do we remove the error message from aria-described by when the error message is removed?

Reply:
Yes we do that, for that I have called this function in  updateValidationMessage, will remove the error message once field is valid